ProcessManager.py

```diff
@@ -52,13 +56,17 @@ def createThread(self):
         ceate one single thread. The thread must then be stored in "self.T".
         """
 
+    def startProcess(self, name):
+        process_dict[name].execute()
```

Comment:
👍 We should probably check if `name` is in `process_dict` before accessing it.

Reply:
Also, if you could add a [docstring](https://www.python.org/dev/peps/pep-0257/) to this method, that would be great.

Reply:
Would the others need a doc string?

Reply:
Yeah, everything need docs. :)